src/main/java/edu/iastate/ece/sd/sdmay2126/runner/selenium/FBASeleniumApplicationExecutor.java

```diff
@@ -16,9 +16,15 @@
 
     @Override
     public void executeApplication(Job job) {
-        new WebDriverWait(driver, Duration.ofSeconds(10))
-                .until(d -> d.findElements(By.cssSelector("button[data-button='runApp']")))
-                .get(2) // FBA is the 3rd application on the page, thus the 3rd run button (0-indexed)
-                .click();
+        try {
+            // Wait until the run button is enabled after a reset.
+            Thread.sleep(2000);
```

Comment:
Note that this will be unnecessary once we scope our selectors to the card, since we can do an `.until(...)` assuming there will be a single match for your selector.